buildSrc/src/main/groovy/nva.search.service.publish-maven.gradle

```diff
@@ -0,0 +1,77 @@
+plugins {
```

Comment:
file for allowing to publish to maven central, maven local, or any other repository